lang/java/avro/src/test/java/org/apache/avro/generic/TestGenericDatumReader.java

```diff
@@ -33,27 +31,9 @@ public class TestGenericDatumReader {
 
   private static final Random r = new Random(System.currentTimeMillis());
 
-  @Test
-  void readerCache() {
```

Comment:
Why do you delete the test ?
Looking at it I think it should still work. You just need to remove the ctor parameter.